bmiapp/lib/pageinfo.dart

```diff
@@ -8,71 +8,70 @@ class Pageinfo extends StatefulWidget {
 }
 
 class PageinfoState extends State {
-  var height = 0.0;
-  var weight = 0.0;
-  var bmi = 0.0;
-  var img = [
-    "https://th.bing.com/th/id/OIP.XJOSqzpR817O9ZPXqgtt2gHafl?w=82&h=349&c=7&r=0&o=5&dpr=1.3&pid=1.7",
-    "https://th.bing.com/th/id/OIP.MaCmnZkgicn7CjbT297tbwHaPR?w=115&h=197&c=7&r=0&o=5&dpr=1.3&pid=1.7",
-    'https://th.bing.com/th/id/OIP.CwWKuANPjNhg2JapdA89VgHaHa?w=194&h=194&c=7&r=0&o=5&dpr=1.3&pid=1.7'
-  ];
-  var level = 0;
-  void bmiupdate() {
-    if (bmi < 18.5) {
-      level = 0;
-    } else if (bmi >= 18.5 && bmi < 25) {
-      level = 1;
-    } else {
-      level = 2;
-    }
-  }
-
+  String? img;
+  double? _bmi;
+  double height = 0;
```

Comment:
as height is in the denominator once the value gets to zero it might lead to infinity to deal with such cases it is used but later a small workaround in the main code made the code to show bmi only when both height and weight are specified , this is a redundant variable now